.github/workflows/build-and-test.yml

```diff
@@ -13,26 +13,32 @@ on:
 jobs:
   spm:
     name: SwiftPM build and test
-    runs-on: macos-latest
```

Comment:
Something wrong with the macOS 14 runners?

Reply:
Pretty sure GitHub doesn't have macOS 14 runners. macOS 13 is still considered beta…